hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneMultipartUploadPartListParts.java

```diff
@@ -77,8 +79,16 @@ public boolean isTruncated() {
     return partInfoList;
   }
 
+  @Deprecated
+  public ReplicationType getReplicationType() {
+    return ReplicationType
+            .fromProto(replicationConfig.getReplicationType());
+  }
+
+  @Deprecated
   public ReplicationFactor getReplicationFactor() {
-    return replicationFactor;
+    return ReplicationFactor
+            .fromProto(ReplicationConfig.getLegacyFactor(replicationConfig));
   }
```

Comment:
Maybe we should add a `getReplicationConfig()` here, as we are deprecating the getType and Factor methods?

Reply:
Sure, I added the getter. It's not yet used, but can be useful for users of the Java API.